core/server/master/src/main/java/alluxio/master/file/DefaultFileSystemMaster.java

```diff
@@ -1066,6 +1066,31 @@ public void listStatus(AlluxioURI path, ListStatusContext context,
     return fileInfos;
   }
 
+  /**
+   * {@link DefaultFileSystemMaster} has two private methods associated with sync metadata:
+   * 1. {@link DefaultFileSystemMaster#loadMetadataIfNotExist} sync metadata in load-only mode,
+   * 2. {@link DefaultFileSystemMaster#syncMetadata} sync metadata not in force mode.
```

Comment:
what is "not in force mode"? what is "force mode"? Can you explain more clearly?

Reply:
That is to say, sync may not execute